server/routes/applications.js

```diff
@@ -302,8 +302,12 @@ router.get('/manage-applications/:appId/:keyType/:keyId/update/:env', (req, res,
     .then(keyData => {
       viewData.this_data.keyName = keyData.name;
       viewData.this_data.keyDescription = keyData.description;
-      viewData.this_data.restrictedIps = keyData.restrictedIPs;
-      viewData.this_data.javaScriptDomains = keyData.jsDomains;
+      if (viewData.this_data.keyType === 'key') {
+        viewData.this_data.restrictedIps = keyData.restrictedIPs;
+        viewData.this_data.javaScriptDomains = keyData.jsDomains;
+      } else if (viewData.this_data.keyName === 'stream') {
+        viewData.this_data.restrictedIps = keyData.restrictedIPs;
+      }
```

Comment:
We could simplify all of this by just putting the key in there directly.

Reply:
I have altered that part now as per your suggestion